collect_app/src/main/java/org/odk/collect/android/tasks/InstanceGoogleSheetsUploader.java

```diff
@@ -71,78 +80,103 @@
     private static final String oauth_fail = "OAUTH Error: ";
     private static final String UPLOADED_MEDIA_URL = "https://drive.google.com/open?id=";
 
-    // needed in case of rate limiting
-    private static final int GOOGLE_SLEEP_TIME = 1000;
-
     private final SheetsHelper sheetsHelper;
     private final DriveHelper driveHelper;
-
     private final GoogleAccountsManager accountsManager;
-    private Integer sheetId;
-    private String sheetName;
     private Outcome outcome;
-    private boolean hasWritePermissionToSheet = false;
+
+    private boolean hasWritePermissionToSheet;
     private boolean authFailed;
-    private String googleSheetsUrl = "";
+
+    private String mainSheetTitle;
+    private String googleSheetsUrl;
     private String spreadsheetId;
+    private String id;
```

Comment:
`id` is only used for the outcome text. Could `jrFormId` be used in the outcome instead so that this field isn't needed?

Reply:
I’ve eliminated it in my proposed changes.